package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "sass": "^1.37.5"
+  }
+}
```

Comment:
This is weird; I did not know npm creates a package.json automatically. I will delete this.